docs/docs/references/users/README.md

```diff
@@ -214,3 +214,69 @@ Updating a user's _custom data_ will overwrite (not merge) its original content
 :::
 
 ### Identities
+
+_Identities_ contain the user info retrieved from social sign-in
+(a.k.a. sign-in with a social connector).
```

Comment:
i.e.

Reply:
should be `contains` as you used _Identites_ as a single subject?

Reply:
updated

Reply:
updated `contains`